.github/workflows/phpunit-tests.yml

```diff
@@ -256,22 +270,37 @@ jobs:
       - name: Install WordPress
         run: npm run env:install
 
+      - name: Run slow PHPUnit tests on PHP 5.6
+        if: ${{ matrix.php == '5.6.20' && matrix.slow56 }}
```

Comment:
:1234: Can these check the slow environment variable only in case other PHP versions are added later.